### PR TITLE
mgr/dashboard: Move force maintenance test to the workflow test suite 

### PR DESCRIPTION
**Pull request by nizamial09**
Fixes: https://tracker.ceph.com/issues/52276
Signed-off-by: Nizamudeen A <nia@redhat.com>

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

File: src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/services.po.ts [hunk 1]
```diff
@@ -24,16 +30,24 @@ export class ServicesPageHelper extends PageHelper {
...
+  clickServiceTab(serviceName: string, tabName: string) {
+    this.getExpandCollapseElement(serviceName).click();
+    cy.get('cd-service-details').within(() => {
+      this.getTab(tabName).click();
+    });
+  }
+
   @PageHelper.restrictTo(pages.create.url)
-  addService(serviceType: string, exist?: boolean) {
+  addService(serviceType: string, exist?: boolean, count = '1') {
```
Comment (avanthakkar) [inline src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/services.po.ts:42]: `count  = '1` : is this to avoid waiting for daemons running?

**Comment by avanthakkar** [comment]
> jenkins test dashboard cephadm

have u rebased the PR to get this changes https://github.com/ceph/ceph/pull/42919?

**Comment by nizamial09** [comment]
> > jenkins test dashboard cephadm
> 
> have u rebased the PR to get this changes #42919?

Yes @avanthakkar The current error is related to the maintenance option. I am trying to debug that. But it seems coming from the cephadm side. There's a tracker I opened https://tracker.ceph.com/issues/52454

File: src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/services.po.ts (hunk 1 above)
Comment (nizamial09) [reply src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/services.po.ts:42]: nope. for testing force maintenance check I need to deploy atleast 2 rgw service. But earlier the count was hardcoded to the code. Now I used a variable to control that. @avanthakkar

**Comment by nizamial09** [comment]
jenkins test dashboard

**Review by avanthakkar — APPROVED** [review]
LGTM!

**Review by pereman2 — APPROVED** [review]

**Pull request merged**